VideoAPI/VideoApi.Services/AudioPlatformService.cs

```diff
@@ -202,6 +202,27 @@ public async Task<AudioPlatformServiceResponse> DeleteAudioStreamAsync(Guid hear
             }
         }
 
+        public async Task<WowzaGetDiagnosticsResponse> GetDiagnosticsAsync()
+        {
+            try
+            {
+                var response = await _wowzaClient.GetDiagnosticsAsync(_configuration.ServerName);
+
+                _logger.LogInformation($"Got Wowza server version for application: {_configuration.ServerName}");
```

Comment:
don't think we to log as it will kick of every 5 mins

Reply:
removed the logging